py3dtilers/Common/loa.py

```diff
@@ -0,0 +1,123 @@
+import os
+from os import listdir
+import sys
+import json
+from shapely.geometry import Point, Polygon
+from ..Common import ObjectToTile, ObjectsToTile, ObjectsToTileWithGeometry
+from ..Common import get_lod1
+
+
+def create_loa(objects_to_tile, loa_path):
+    return group_features_by_polygons(objects_to_tile, loa_path)
+
+
+def group_features_by_polygons(features, path):
```

Comment:
What are the design reasons behind the fact that this files holds a set of functions
as opposed to a class (or as set of classes) ?